docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-to-kubernetes-version-1.25.md

```diff
@@ -32,3 +32,17 @@ Grafana Mimir does not require any special permissions on the hosts that it
 runs on. Because of this, you can deploy it in environments that enforce the
 Kubernetes [Restricted security policy](https://kubernetes.io/docs/concepts/security/pod-security-standards/).
 {{% /admonition %}}
+
+## Troubleshoot
+
+If you have upgraded to Kubernetes 1.25 and see the following error containing PodSecurityPolicy during a Helm release upgrade:
+
+```
+resource mapping not found for name: "mimir" namespace: "" from "":
+no matches for kind "PodSecurityPolicy" in version "policy/v1beta1" ensure CRDs are installed first
+```
+
+1. Optionally follow the Kubernetes [Migrate from PodSecurityPolicy to the Built-In PodSecurity Admission Controller](https://kubernetes.io/docs/tasks/configure-pod-container/migrate-from-psp/) guide to replace PodSecurityPolicy.
+1. Set the `rbac.create` value to `false`.
```

Comment:
That shouldn't be necessary because the chart already doesn't render it

Reply:
this way you can reuse this doc on older chart versions!